src/Uno.UI.Wasm/ts/Windows/Devices/Midi/MidiInPort.ts

```diff
@@ -0,0 +1,52 @@
+﻿namespace Windows.Devices.Midi {
+	export class MidiInPort {
+		private static dispatchMessage:
+			(instanceId: string, serializedMessage: string, timestamp: number) => number;
+
+		private static instanceMap: { [managedId: string]: MidiInPort };
+
+		private managedId: string;
+		private inputPort: WebMidi.MIDIInput;
+
+		private constructor(managedId: string, inputPort: WebMidi.MIDIInput) {
+			this.managedId = managedId;
+			this.inputPort = inputPort;
+		}
+
+		public static createPort(managedId: string, encodedDeviceId: string) {
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			const deviceId = decodeURIComponent(encodedDeviceId);
+			var input = midi.inputs.get(deviceId);
+			MidiInPort.instanceMap[managedId] = new MidiInPort(managedId, input);
+		}
+
+		public static removePort(managedId: string) {
+			var instance = MidiInPort.instanceMap[managedId];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'instance' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)